srcpkgs/duckdb/template

```diff
@@ -0,0 +1,57 @@
+# Template file for 'duckdb'
+pkgname=duckdb
+version=0.10.1
+revision=1
+archs="x86_64* aarch64*" # 32-bit FTBFS
+build_style=cmake
+build_helper="python3"
+configure_args="-DOVERRIDE_GIT_DESCRIBE=v${version}-0-gdeadbeeff
```

Comment:
just `v${version}` works here

Reply:
yeah but they add [random stuff](https://github.com/duckdb/duckdb/blob/v0.10.1/CMakeLists.txt#L248-L259) anyway
clearly the [python counterpart](https://github.com/duckdb/duckdb/blob/v0.10.1/tools/pythonpkg/setup.py#L395) was written by someone else :smile: